src/EPR.PRN.Backend.Obligation.UnitTests/Strategies/GeneralCalculationStrategyTests.cs

```diff
@@ -3,115 +3,116 @@
 using EPR.PRN.Backend.Obligation.Interfaces;
 using EPR.PRN.Backend.Obligation.Strategies;
 using Moq;
+using System.Diagnostics.CodeAnalysis;
 
-namespace EPR.PRN.Backend.Obligation.UnitTests.Strategies
+namespace EPR.PRN.Backend.Obligation.UnitTests.Strategies;
+
+[ExcludeFromCodeCoverage]
```

Comment:
remove

Reply:
We don't need to look for code coverage on unit tests